Tests/TestingApps/PaywallsTester/PaywallsTester/Views/AppContentView.swift

```diff
@@ -52,7 +47,7 @@ struct AppContentView: View {
                 }
             #endif
 
-            if self.isPurchasesConfigured {
+            if Purchases.isConfigured {
```

Comment:
This broke the sample app when not configuring the API key(s). Before this change, the app would just show the `Examples` tab, but now it looks like it's trying to configure `Purchases` with an empty API key and it shows everything.